modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/DepotEntryVerticalCardPropsTransformer.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import confirmDepotEntryDeletion from './confirmDepotEntryDeletion.es';
+
+const ACTIONS = {
+	deleteDepotEntry(itemData) {
+		if (confirmDepotEntryDeletion()) {
+			submitForm(document.hrefFm, itemData.deleteDepotEntryURL);
+		}
+	},
+
+	permissionsDepotEntry(itemData) {
+		Liferay.Util.openWindow({
```

Comment:
We can use `openModal` here, imported from `frontend-js-web`.
```
openModal({
    title: Liferay.Language.get('permissions'),
    url: itemData.permissionsDepotEntryURL,
})
```

Reply:
I knew there was something we could import I just couldn't remember the name :+1:  :rofl: